programs/bpf/tests/programs.rs

```diff
@@ -3524,3 +3526,120 @@ fn test_program_fees() {
     let post_balance = bank_client.get_balance(&mint_keypair.pubkey()).unwrap();
     assert_eq!(pre_balance - post_balance, expected_min_fee);
 }
+
+#[test]
+#[cfg(feature = "bpf_rust")]
+fn test_get_minimum_delegation() {
+    solana_logger::setup();
+
+    let GenesisConfigInfo {
+        genesis_config,
+        mint_keypair,
+        ..
+    } = create_genesis_config(100_123_456_789);
+    let mut bank = Bank::new_for_tests(&genesis_config);
+    bank.feature_set = Arc::new(FeatureSet::all_enabled());
+
+    let (name, id, entrypoint) = solana_bpf_loader_program!();
+    bank.add_builtin(&name, &id, entrypoint);
+    let bank = Arc::new(bank);
+    let bank_client = BankClient::new_shared(&bank);
+
+    let program_id = load_bpf_program(
+        &bank_client,
+        &bpf_loader::id(),
+        &mint_keypair,
+        "solana_bpf_rust_get_minimum_delegation_data",
+    );
+
+    let stake_keypair = Keypair::new();
+    let stake_account = AccountSharedData::new(123_456_789, 0, &stake::program::id());
+    bank.store_account(&stake_keypair.pubkey(), &stake_account);
```

Comment:
Do you need a stake account? I thought you just needed to invoke the stake program to get the min balance

Reply:
This is also what I'm trying to figure out 😊

Reply:
Ok yeah there's no reason for it